tensorflow/compiler/tf2xla/cpu_function_runtime.h

```diff
@@ -104,7 +104,7 @@ class BufferInfo {
  private:
   BufferInfo() = default;
 
-  enum class Kind : unsigned {
+  enum class Kind : uint64 {
```

Comment:
It must be `uint64` because MSVC doesn't merge bit-fields of different size. I. e. `sizeof(unsigned)` is `4` and `sizeof(uint64)` is `8`. When they get combined, `sizeof(BufferInfo)` will be `24` different from `16` expected by the `assert`.

Reply:
I meant, literally please add a code comment that says:

```
// The enum class kind must be uint64 because ...
```

Reply:
Sorry I didn't unterstand. Fixed.